django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -21,6 +26,10 @@
             fontSize: 14,
             showInvisibles: true
         });
+
+        //add  listener to .screen-mode button for toggleScreenMode
```

Comment:
leave a space after `//`:

`// add  listener to .screen-mode button for toggleScreenMode`